app/initializers/ember-i18n-cp-validations.js

```diff
@@ -16,7 +16,7 @@ const {
 } = Ember;
 
 function isSafeString(input) {
-  return (typeof isHTMLSafe === 'function' && isHTMLSafe(input)) || (input instanceof Handlebars.SafeString);
+  return (typeof isHTMLSafe === 'function' && isHTMLSafe(input)) || (input && typeof input.toHTML === 'function');
```

Comment:
Yeah. I see that the code is being used in the `isHTMLSafe` function itself inside ember https://github.com/emberjs/ember.js/blob/aeeddf7c00b42a340494465252481b5414d87009/packages/ember-glimmer/lib/utils/string.js

So the changes in my commit are actually duplicated. I guess the best way would be to just use the isHTMLSafe function for ember versions that support it, and only use `instanceof SafeString` in older ember versions that will not throw a deprecation message.